app/templates/components/user-card.hbs

```diff
@@ -2,13 +2,12 @@
 
 <div class = "card-container">
     {{user-avatar title=firstname imm=image}}
-    <div class = "namebox">
+    <div class = "font-style namebox">
          {{firstname}} {{lastname}}
     </div>
 
-    <div class = "emailbox">
+    <div class = "font-style emailbox">
```

Comment:
`font-style` class needn't to be added in all the places. We can apply this to any common parent i.e (body).